Plugin/Tax/Config/ShippingTaxPlugin.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FireGento\MageSetup\Plugin\Tax\Config;
+
+use FireGento\MageSetup\Model\Config as FireGentoConfig;
+use Magento\Checkout\Model\Cart;
+use Magento\Customer\Model\ResourceModel\GroupRepository;
+use Magento\Customer\Model\Session;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Quote\Model\Quote\Item;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Store\Model\Store;
+use Magento\Tax\Model\Calculation\Proxy;
+use Magento\Tax\Model\Config;
+
+class ShippingTaxPlugin
+{
+    public const CONFIG_PATH_DYNAMIC_SHIPPING_TAX_CLASS = 'tax/classes/dynamic_shipping_tax_class';
```

Comment:
Please add a new method in the `Config` model for retrieving this configuration and use it.